### PR TITLE
Decimals in generator delay value

### LINKED ISSUE #247: Decimal Value For Delay In generators.yml
I want to make a config so that some arenas have different spawning rate like hypixel.

```
Default:
  iron:
    delay: 0.33
    amount: 1
    spawn-limit: 48
```
I'm pretty sure that this values won't work and still spawn 1 iron ingot per second.
So I hope you could make it so we could set a value like this and get more than 1 ingot per second

Thank You and I hope this would be added :........)

### PR DESCRIPTION
**Pull request by reussy**
Allows using decimals in the generator delay.
For avoid visual bugs I used Math#round in {seconds} placeholder.

Closes #247

### CODE REVIEW

File: bedwars-plugin/src/main/java/com/andrei1058/bedwars/arena/OreGenerator.java [hunk 1]
```diff
@@ -93,7 +94,7 @@ public void upgrade() {
             case DIAMOND:
                 upgradeStage++;
                 if (upgradeStage == 2) {
-                    delay = getGeneratorsCfg().getInt(getGeneratorsCfg().getYml().get(arena.getGroup() + "." + ConfigPath.GENERATOR_DIAMOND_TIER_II_DELAY) == null ?
+                    delay =  getGeneratorsCfg().getInt(getGeneratorsCfg().getYml().get(arena.getGroup() + "." + ConfigPath.GENERATOR_DIAMOND_TIER_II_DELAY) == null ?
```
Comment (ajgeiss0702) [inline bedwars-plugin/src/main/java/com/andrei1058/bedwars/arena/OreGenerator.java]: was this accidental?

**Review by ajgeiss0702 — CHANGES_REQUESTED** [review]
From what I can see this will just round the delay, but wont actually use the decimal value

File: bedwars-plugin/src/main/java/com/andrei1058/bedwars/arena/OreGenerator.java (hunk 1 above)
Comment (reussy) [reply bedwars-plugin/src/main/java/com/andrei1058/bedwars/arena/OreGenerator.java:97]: yes, but idk why it was sent this way

Comment (reussy) [reply bedwars-plugin/src/main/java/com/andrei1058/bedwars/arena/OreGenerator.java]: > 

yes, but idk why it was sent this way

**Comment by reussy** [comment]
> From what I can see this will just round the delay, but wont actually use the decimal value
It rounds it for holograms but the value (decimal or integer) is used when spawning a new ore

**Comment by ajgeiss0702** [comment]
But from what I see you are rounding that value too

**Comment by andrei1058** [comment]
generators do not support doubles for now

**Pull request closed without merging**